python/test/t_QuantileConfidence_std.py

```
@@ -3,468 +3,736 @@
+    alpha = 0.95
+    beta = 0.90
+    algo = ot.QuantileConfidence(alpha, beta)
+    size = 100
+    distribution = ot.Normal()
+    quantile_alpha = distribution.computeQuantile(alpha)
+    # This should be as large as possible (but is limited by CPU time)
+    number_of_repetitions = 3000
+    count_upper = 0
+    count_lower = 0
+    count_bilateral = 0
+    for r in range(number_of_repetitions):
```

Ruff (0.15.13)

[warning] Loop control variable `r` not used within loop body

Rename unused `r` to `_r`

(B007)